### PR TITLE
Mvc models

### PR DESCRIPTION
**Pull request by Casanova569**
Summary:
Objetive: The trello card is considered approved, if it completes the configuration of the Database and if the Movie Model was generated.

Trello Card:
Name: MVC - Models
Link: https://trello.com/c/aKwxQCg0/4-mvc-models

### CODE REVIEW

File: src/Models/MvcMovieContext.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    public class MvcMovieContext : DbContext
+    {
+        public MvcMovieContext (DbContextOptions<MvcMovieContext> options)
+            : base(options)
+        {
+        }
+
+        public DbSet<MvcMovie.Models.Movie> Movie { get; set; }
+    }
+}
```
Comment (mikolaslorant) [inline src/Models/MvcMovieContext.cs]: Add EOF (end of file blank line)

File: src/Startup.cs [hunk 2]
```diff
@@ -34,17 +36,15 @@ public void ConfigureServices(IServiceCollection services)
                 options.MinimumSameSitePolicy = SameSiteMode.None;
             });
 
-            //services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
-            services.AddMvc();
+            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+            //services.AddMvc();
```
Comment (mikolaslorant) [inline src/Startup.cs]: remove comments

File: src/Models/MvcMovieContext.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+
+namespace MvcMovie.Models
+{
+    public class MvcMovieContext : DbContext
+    {
+        public MvcMovieContext (DbContextOptions<MvcMovieContext> options)
```
Comment (mikolaslorant) [inline src/Models/MvcMovieContext.cs]: when a method is empty try to write everything in same line. Eg.:
public DatabaseContext (DbContextOptions<DatabaseContext> options) {}

File: src/appsettings.Development.json [hunk 4]
```diff
@@ -5,5 +5,6 @@
       "System": "Information",
       "Microsoft": "Information"
     }
-  }
```
Comment (mikolaslorant) [inline src/appsettings.Development.json:8]: Add appsettings.Development.json to your .gitignore file. Your development json should be private and not shared with the team using the repository.

File: src/appsettings.Development.json [hunk 5]
```diff
@@ -5,5 +5,6 @@
       "System": "Information",
       "Microsoft": "Information"
     }
-  }
+  },
+  "ConnectionString": "User ID=dev;Password=dev;Host=localhost;Port=5432;Database=MvcMovie.db;Pooling=true;Timeout=1024;CommandTimeout=1024;"
```
Comment (mikolaslorant) [inline src/appsettings.Development.json]: Eg.: this connection String is particular to your local configuration

File: src/Models/MvcMovieContext.cs [hunk 6]
```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+
+namespace MvcMovie.Models
+{
+    public class MvcMovieContext : DbContext
```
Comment (mikolaslorant) [inline src/Models/MvcMovieContext.cs]: Although MvcMovie corresponds with the name of the application I suggest using a more generic name like DatabaseContext for the class. (just a suggestion)

**Review by mikolaslorant — CHANGES_REQUESTED** [review]

File: src/appsettings.Development.json (hunk 4 above)
Comment (mikolaslorant) [reply src/appsettings.Development.json:8]: remove this file from your pull request and set it in your .gitignore

**Review by mikolaslorant — CHANGES_REQUESTED** [review]

File: src/Models/MvcMovieContext.cs [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+    {
+        public MvcMovieContext (DbContextOptions<MvcMovieContext> options): base(options){}
+        
+        public DbSet<MvcMovie.Models.Movie> Movie { get; set; }
```
Comment (mikolaslorant) [inline src/Models/MvcMovieContext.cs]: add using at the top of file to import models and make Movie plural like so:
public DbSet<Movie> Movies { get; set; }

Comment (mikolaslorant) [reply src/Models/MvcMovieContext.cs]: setting movies to plural sets the Table name in postgres

**Review by mikolaslorant — CHANGES_REQUESTED** [review]

File: src/Models/MvcMovieContext.cs (hunk 7 above)
Comment (truccomarcos) [reply src/Models/MvcMovieContext.cs]: @Casanova569 don't forget to update this. 
`public DbSet<Movie> Movies { get; set; }`
You might need to run another migration afterwards

File: src/appsettings.json [hunk 8]
```diff
@@ -4,5 +4,8 @@
       "Default": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "MovieContext": "Data Source=MvcMovie.db"
```
Comment (truccomarcos) [inline src/appsettings.json]: Why is this connection string here?

**Review by truccomarcos — CHANGES_REQUESTED** [review]

**Review by truccomarcos — APPROVED** [review]

**Pull request merged**